honeybadgermpc/secretshare_functionality.py

```diff
@@ -50,38 +50,38 @@ def __init__(self, sid, N, f, inputFromDealer=None):
             self.outputs[i].set_result(share)
 
 
-def SecretShare_IdealProtocol(N, f):
-    class SecretShare_IdealProtocol(object):
+def secret_share_ideal_protocol(n, f):
+    class SecretShareIdealProtocol(object):
         _instances = {}     # mapping from (sid,myid) to functionality shared state
 
-        def __init__(self, sid, Dealer, myid):
+        def __init__(self, sid, dealer, myid):
             self.sid = sid
-            self.Dealer = Dealer
+            self.Dealer = dealer
             # Create the ideal functionality if not already present
-            if sid not in SecretShare_IdealProtocol._instances:
-                SecretShare_IdealProtocol._instances[sid] = \
-                    SecretShare_Functionality(
-                        sid, N, f, inputFromDealer=asyncio.Future())
-            F_SecretShare = SecretShare_IdealProtocol._instances[sid]
+            if sid not in SecretShareIdealProtocol._instances:
+                SecretShareIdealProtocol._instances[sid] = \
+                    SecretShareFunctionality(
+                        sid, n, f, input_from_dealer=asyncio.Future())
+            f_secret_share = SecretShareIdealProtocol._instances[sid]
 
             # If dealer, then provide input
-            if myid == Dealer:
-                self.inputFromDealer = F_SecretShare.inputFromDealer
+            if myid == dealer:
+                self.inputFromDealer = f_secret_share.inputFromDealer
             else:
                 self.inputFromDealer = None
 
             # A future representing the output is available
-            self.output = F_SecretShare.outputs[myid]
-    return SecretShare_IdealProtocol
+            self.output = f_secret_share.outputs[myid]
```

Comment:
This doesn't look right - ideal functionalities in the paper are always indicated by a capital F. A lowercase f doesn't mean the same thing. If lowercase is essential, then it should be replaced by `functionality_` or `func_`

Reply:
pep8 doesn't permit uppercase, I will change it to `func`.